.github/dependabot.yml

```diff
@@ -0,0 +1,12 @@
+# To get started with Dependabot version updates, you'll need to specify which
```

Comment:
question: what does this do?

Something to do with the auto-pr shenanigans from github?

Reply:
Removed; didn't mean for this to be on my fork of master - but it basically makes dependabot pr to the develop branch instead of master to make it "safer" to test and merge back.